deployment/scripts/logstash.properties

```diff
@@ -2,9 +2,9 @@
 Operator.level=INFO
 handlers=java.util.logging.ConsoleHandler,java.util.logging.FileHandler
 java.util.logging.ConsoleHandler.level=INFO
-java.util.logging.ConsoleHandler.formatter=oracle.kubernetes.operator.logging.LoggingFormatter
+java.util.logging.ConsoleHandler.formatter=oracle.kubernetes.operator.logging.OperatorLoggingFormatter
```

Comment:
Should this be a specific class for the operator vs. the webhook?

Reply:
I'm not sure if we need a specific class for the operator vs. the webhook. If so, I can look into it next week.